process/smartContract/processorV2/processV2.go

```diff
@@ -603,6 +603,9 @@ func (sc *scProcessor) updateDeveloperRewards(
 	}
 
 	moveBalanceGasLimit := sc.economicsFee.ComputeGasLimit(tx)
+	if common.IsValidRelayedTxV3(tx) {
```

Comment:
I don't think we need this, as the moveBalanceGasLimit completely needs to not be considered for dev fees, not only part of it.

From what I understand, here you are removing the extra part for relayer signature validation from the base cost, and will be using that instead for dev fees (by not removing it from the usedGasByMainSC).

You will also need to consider this for the preparation for SC execution, there the gas limit you provide in vmInput needs to not contain this part.

Reply:
removed as discussed